browser/ui/webui/brave_rewards_page_ui.cc

```diff
@@ -1194,66 +1187,69 @@ void RewardsDOMHandler::OnGetAdsHistory(const base::ListValue& ads_history) {
 }
 
 void RewardsDOMHandler::ToggleAdThumbUp(const base::ListValue* args) {
-  CHECK_EQ(3U, args->GetSize());
+  CHECK_EQ(1U, args->GetSize());
+
   if (!ads_service_) {
     return;
   }
 
   AllowJavascript();
 
-  const std::string id = args->GetList()[0].GetString();
-  const std::string creative_set_id = args->GetList()[1].GetString();
-  const int action = args->GetList()[2].GetInt();
+  ads::AdContentInfo ad_content;
+  const base::Value value = args->GetList()[0].Clone();
```

Comment:
Clone() call is not needed here. We can use here a const reference:
`const base::Value& value = args->GetList()[0];`

or pass `args->GetList()[0]` directly to `ad_content.FromValue`

Reply:
Fixed, thanks